# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/efivar/hotfix-no-mandoc.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- efivar-38/Makefile.vanilla	2022-04-01 13:27:57.923829735 +0200
+++ efivar-38/Makefile	2022-04-01 13:28:11.193830420 +0200
@@ -7,7 +7,7 @@
 include $(TOPDIR)/src/include/coverity.mk
 include $(TOPDIR)/src/include/scan-build.mk
 
-SUBDIRS := src docs
+SUBDIRS := src # docs
 
 all : | efivar.spec src/include/version.mk prep
 all clean install prep :
